app/models/outcomes_dashboard.rb

```diff
@@ -0,0 +1,19 @@
+class OutcomesDashboard
+  def initialize(courses)
+    @courses = courses
+  end
+
+  def courses_without_outcomes
+    courses.where(outcomes_count: 0)
+  end
+
+  def unaligned_courses
+    @unaligned_courses ||= courses.with_custom_outcomes.select do |course|
+      StandardOutcome.unaligned_with(course).present?
```

Comment:
If you use `any?` instead of `present?`, ActiveRecord magic will turn this into a `count` instead of `select *`.